public/fonts.css

```diff
@@ -72,127 +68,3 @@
     U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215,
     U+FEFF, U+FFFD;
 }
-
-.material-icons {
-  font-family: 'Material Icons';
```

Comment:
Codacy found an issue: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/gh/feugy/melodie/pullRequest?prid=6399353)